pom.xml

```diff
@@ -713,13 +713,12 @@
       </build>
     </profile>
 
-    <!-- Profile for CI tests to have less output -->
+    <!-- Profile for tests to have more output -->
     <profile>
-      <id>ci-test</id>
+      <id>verbose-test</id>
```

Comment:
What about keeping the `ci-test` profile to be verbose as is, and add a new compact one which is activated by default?

Reply:
As of now our output is verbose by default, with the `ci-test` profile being more compact. I tried looking into enabling `ci-test` by default, but when a profile is `activeByDefault`, it gets deactivated whenever another profile becomes active. The problem then is that `parquet-mr` has a few profiles that get activated automatically depending on the environment Maven is run in, so we cannot rely on it in that way. Because of this, the PR gets rid of the slimmed down `ci-test` profile and replaces it with a verbose one, such that the (currently default) verbose output becomes opt-in, with the more lean output just being the default setting.

Reply:
Sounds good to me. 

@gszadovszky @shangxinli Do you have different opinions?

Reply:
I think `verbose-test` is a better name than `ci-test`